tdrs-frontend/src/components/UploadReport/UploadReport.jsx

```diff
@@ -6,13 +6,20 @@ import { fileInput } from 'uswds/src/js/components'
 import Button from '../Button'
 
 import FileUpload from '../FileUpload'
+import useEventLogger from '../../utils/eventLogger'
 
 function UploadReport({ handleCancel }) {
   // The currently selected year from the reportingYears dropdown
   const selectedYear = useSelector((state) => state.reports.year)
 
-  // Ensure newly rendered header is focused, else it won't be read be screen readers.
+  // Ensure newly rendered header is focused,
+  // else it won't be read be screen readers.
   const headerRef = useRef(null)
+  const logger = useEventLogger()
+
+  const onSubmit = () => {
```

Comment:
Log user File Upload submissions

Reply:
Note that this will be expanded on in another issue when we begin login upload and download activity #762